deployments/operator/crd/bases/deviceplugin.intel.com_dlbdeviceplugins.yaml

```diff
@@ -72,6 +72,45 @@ spec:
                 description: NodeSelector provides a simple way to constrain device
                   plugin pods to nodes with particular labels.
                 type: object
+              tolerations:
+                description: Tolerations for the plugin Pod
+                items:
+                  description: |-
+                    The pod this Toleration is attached to tolerates any taint that matches
+                    the triple <key,value,effect> using the matching operator <operator>.
+                  properties:
+                    effect:
+                      description: |-
+                        Effect indicates the taint effect to match. Empty means match all taint effects.
+                        When specified, allowed values are NoSchedule, PreferNoSchedule and NoExecute.
+                      type: string
+                    key:
+                      description: |-
+                        Key is the taint key that the toleration applies to. Empty means match all taint keys.
+                        If the key is empty, operator must be Exists; this combination means to match all values and all keys.
+                      type: string
+                    operator:
+                      description: |-
+                        Operator represents a key's relationship to the value.
+                        Valid operators are Exists and Equal. Defaults to Equal.
+                        Exists is equivalent to wildcard for value, so that a pod can
+                        tolerate all taints of a particular category.
+                      type: string
+                    tolerationSeconds:
```

Comment:
Other items do not use `toleration` prefix, why it's used here?

(Same comment goes for all other plugins' CRDs.)

Reply:
These CRDs are derived from the golang structs. `tolerationSeconds` is the name of the variable in the struct which comes from k8s component that I can't change.